packages/amplify-e2e-tests/src/__tests__/auth_2.test.ts

```diff
@@ -120,10 +122,18 @@ describe('amplify add auth...', () => {
     const functionName = `${authKey}PostConfirmation-integtest`;
     const authMeta = meta.auth[authKey];
     const id = authMeta.output.UserPoolId;
-    const userPool = await getUserPool(id, meta.providers.awscloudformation.Region);
+    const region = meta.providers.awscloudformation.Region;
```

Comment:
enhance the test for deny cases with different user and group assignment for better coverage.